docs/en/reference/working-with-objects.rst

```diff
@@ -326,46 +326,36 @@ in multiple ways with very different performance impacts.
 Detaching entities
 ------------------
 
-An entity is detached from an EntityManager and thus no longer
-managed by invoking the ``EntityManager#detach($entity)`` method on
-it or by cascading the detach operation to it. Changes made to the
-detached entity, if any (including removal of the entity), will not
-be synchronized to the database after the entity has been
+All entities are detached from an EntityManager and thus no longer
+managed by it after invoking the ``EntityManager#clear()`` method.
+Changes made to the detached entities, if any (including their removal),
+will not be synchronized to the database after they have been
 detached.
 
-Doctrine will not hold on to any references to a detached entity.
+Doctrine will not hold on to any references to detached entities.
 
 Example:
 
 .. code-block:: php
 
     <?php
-    $em->detach($entity);
+    $em->clear();
 
 The semantics of the detach operation, applied to an entity X are
 as follows:
 
-
 -  If X is a managed entity, the detach operation causes it to
-   become detached. The detach operation is cascaded to entities
-   referenced by X, if the relationships from X to these other
-   entities is mapped with cascade=DETACH or cascade=ALL (see
-   ":ref:`Transitive Persistence <transitive-persistence>`"). Entities which previously referenced X
+   become detached. Entities which previously referenced X
```

Comment:
I think this is about cascades. The `Entities which previously referenced X will continue to reference X.` is correct though.
Indeed, needs rephrasing